README_new.md

```diff
@@ -0,0 +1,617 @@
+# Orion Context Broker
```

Comment:
To make easir the review, avoding a very noisy diff in the PR. But the idea is to use only README.md at the end, note from PR body:

> After LGTM this PR:
> *    Will replace README.md by README_new.md
> *    Will be squashed to only one commit.